app/views/receipt_mailer/receipt_email.html.erb

```diff
@@ -0,0 +1,22 @@
+<p>Hello,</p>
+
+<p>You are receiving this message to confirm the successful submission of the
+electronic version of your <%= @thesis.graduation_month %> <%=
+@thesis.graduation_year %> thesis "<%= @thesis.title %>" to the MIT Libraries.
+</p>
+
+<p>You do not need to reply to this message.</p>
+
+<p>Theses will be added to DSpace after degrees have been awarded and the thesis
+has been fully processed by the MIT Libraries (typically 3-6 months after the
+degree date).</p>
+
+<p>If you require further assistance, please send an email to
+etheses-admin@mit.edu.</p>
+
+<p>Thank you,<br />
+MIT Libraries</p>
+
+<p>We would like your feedback!<br />
+Please take a moment to fill out a short survey about this service:<br />
+<a href="https://goo.gl/QwE7ay">https://goo.gl/QwE7ay</a></p>
```

Comment:
Do we have any way whatsoever to monitor for linkrot?

Reply:
no

Reply:
🙈